alignak/action.py

```diff
@@ -101,41 +102,48 @@ def no_block_read(output):
         return ''
 
 
-class ActionBase(object):
+class ActionBase(Item):
     """
     This abstract class is used just for having a common id for both
     actions and checks.
     """
-    _id = 0
     process = None
 
     properties = {
         'is_a':             StringProp(default=''),
         'type':             StringProp(default=''),
+        'creation_time':       FloatProp(default=0.0),
         '_in_timeout':      BoolProp(default=False),
-        'status':           StringProp(default=''),
+        'status':           StringProp(default='scheduled'),
         'exit_status':      IntegerProp(default=3),
-        'output':           StringProp(default=''),
-        't_to_go':          FloatProp(default=0),
+        'output':           StringProp(default='', fill_brok=['full_status']),
+        't_to_go':          FloatProp(default=0.0),
         'check_time':       IntegerProp(default=0),
         'execution_time':   FloatProp(default=0.0),
         'u_time':           FloatProp(default=0.0),
         's_time':           FloatProp(default=0.0),
         'reactionner_tag':  StringProp(default='None'),
         'env':              DictProp(default={}),
-        'module_type':      StringProp(default='fork'),
-        'worker':           StringProp(default='none')
+        'module_type':      StringProp(default='fork', fill_brok=['full_status']),
+        'worker':           StringProp(default='none'),
+        'command':          StringProp(),
+        'timeout':          IntegerProp(default=10),
+        'ref':              StringProp(default=''),
     }
 
-    @staticmethod
-    def assume_at_least_id(_id):
-        """Set Action._id to the maximum of itself and _id
+    def __init__(self, params=None):
+        super(ActionBase, self).__init__(params)
+        self.creation_time = time.time()
+        self.fill_default()
 
-        :param _id: action id to compare (from a previous run usually)
-        :type _id: int
-        :return: None
+    def get_id(self):
```

Comment:
Perhaps simplify and remove this function and update the 2 lines call this function and use object.uuid instead object.get_id()